### PR TITLE
feat: adapt plugin for ingester

### PR DESCRIPTION
**Pull request by nacknime-official**

### CODE REVIEW

File: config/geyser-plugin-config.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "libpath": "../target/release/libsolana_geyser_plugin_scaffold.so",
```
Comment (wedancedalot) [inline config/geyser-plugin-config.json:2]: rollback this

File: Cargo.toml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 [package]
-name = "solana-geyser-plugin-scaffold"
```
Comment (wedancedalot) [inline Cargo.toml:2]: rollback this

File: src/receiver.rs [hunk 3]
```diff
@@ -27,37 +33,55 @@ impl TcpReceiver {
...
+        //
+        // Example:
+        // let std_stream = std::net::TcpStream::connect_timeout(&addr, self.connect_timeout)?;
+        // std_stream.set_nonblocking(true)?;
+        // let stream = TcpStream::from_std(std_stream)?;
+        //
+        // Another approach is to use tokio::time::timeout like here
+        // https://stackoverflow.com/a/63465533
+        //
+        // I didn't test any of those approaches
```
Comment (wedancedalot) [inline src/receiver.rs:61]: What is your suggestion for this?

File: Cargo.toml (hunk 2 above)
Comment (nacknime-official) [reply Cargo.toml:2]: the version rollback too?

**Pull request merged**